### ISSUE TITLE
Backtick after uppercase letter is smaller

### ISSUE THREAD

**Issue by simeonschaub** [cormullion/juliamono#8]
If there is an uppercase letter in front of a backtick, the backtick gets smaller than usual:

![Screenshot from 2020-08-02 23-06-58](https://user-images.githubusercontent.com/5220528/89132490-e23e0900-d514-11ea-844d-74260c90dec4.png)

I'm guessing this was not intentional, but I also don't really understand why that would happen.

**Comment by cormullion** [comment]
That's a nice one1 - I tracked it down to a lower-case grave accent that hasn't been set properly:

```julia> s = "H`"
"H`"

julia> lowercase(s)
"h`"
```

(Today I learnt about lower-case grave accents... )

I will try and fix it some time later this month, when I get back to it...

**Comment by JeffBezanson** [comment]
Interesting; I have not seen this.

**Comment by cormullion** [comment]
As a temporary workaround, you can switch off case-sensitive punctuation in some text environments (eg Atom and probably VSCode) with something like:

```
font-feature-settings: "case" on       etc
``` 

This will give you the 'wrong' smaller uppercase grave accent all the time. (You'll also get a different dieresis (0xa8) and acute (0xb4) but they're probably not too bad for now...)

**Comment by simeonschaub** [comment]
Fixed in 0.005! :tada:
